public/admin/bower_components/bootstrap-social/.editorconfig

```diff
@@ -0,0 +1,11 @@
+# editorconfig.org
```

Comment:
Are you sure this file should be part of the project? It only seems related to your editor.